config/routes.rb

```diff
@@ -1,5 +1,8 @@
 Rails.application.routes.draw do
-  root to: "pages#home"
+  devise_for :users
+  root to: 'ships#index'
+  resources :ships, only: %i[index new create]
```

Comment:
As the ships are our object, we need all 7 routes for them. Index, show, new, create, edit, update, destroy.

Reply:
We will also need some of the bookings routes, not all of them, yet to discuss.